build/patches/com_github_bazelbuild_rules_go.patch

```diff
@@ -0,0 +1,25 @@
+commit 7ca0f1924b85a139190a3bc6fc5f5cc5ea5d6eb6
+Author: Rail Aliiev <rail@iqchoice.com>
+Date:   Wed Nov 10 09:44:34 2021 -0500
+
+    Save test output in test2json format
+    
+    If `GO_TEST_JSON_OUTPUT_FILE` environment variable is set, write the
+    test output in test2json format.
+
+diff --git a/go/tools/bzltestutil/wrap.go b/go/tools/bzltestutil/wrap.go
+index 382d636a..cbb2e7e6 100644
+--- a/go/tools/bzltestutil/wrap.go
++++ b/go/tools/bzltestutil/wrap.go
+@@ -85,6 +85,11 @@ func Wrap(pkg string) error {
+ 			return fmt.Errorf("error while generating testreport: %s", werr)
+ 		}
+ 	}
++	if out, ok := os.LookupEnv("GO_TEST_JSON_OUTPUT_FILE"); ok {
```

Comment:
We already have a fork of `rules_go` with some patches applied, can you just push this patch to our repo instead? We update `rules_go` not infrequently, and keeping this patch file up-to-date is going to be way more complicated than using Git.

1. Check out `release-0.26` of `cockroachdb/rules_go` (should be commit `4a7bcc9b9051eb6a12bcb03a9bf38138c4bbc2ea`)
2. Push this commit to `release-0.26`
3. Download the new tarball: `github.com/cockroachdb/rules_go/tarball/$NEW_COMMIT_ID`
4. Mirror the file to `https://storage.googleapis.com/public-bazel-artifacts/bazel/` and update `WORKSPACE` accordingly. There shouldn't be a need to apply the patch to `DEPS.bzl` so you can leave it untouched.

Reply:
Good point. Probably I can just use my own fork for faster iterations instead of using this method.